src/assert.rs

```diff
@@ -350,25 +353,18 @@ impl Assert {
             if expect_success != output.status.success() {
                 let out = String::from_utf8_lossy(&output.stdout).to_string();
                 let err = String::from_utf8_lossy(&output.stderr).to_string();
-                bail!(ErrorKind::StatusMismatch(
-                    self.cmd.clone(),
-                    expect_success,
-                    out,
-                    err,
-                ));
+                bail!(ErrorKind::StatusMismatch(self.cmd.clone(), expect_success, out, err));
             }
         }
 
         if self.expect_exit_code.is_some() && self.expect_exit_code != output.status.code() {
             let out = String::from_utf8_lossy(&output.stdout).to_string();
             let err = String::from_utf8_lossy(&output.stderr).to_string();
-            bail!(ErrorKind::ExitCodeMismatch(
-                self.cmd.clone(),
-                self.expect_exit_code,
-                output.status.code(),
-                out,
-                err,
-            ));
+            bail!(ErrorKind::ExitCodeMismatch(self.cmd.clone(),
+                                              self.expect_exit_code,
+                                              output.status.code(),
+                                              out,
+                                              err));
```

Comment:
rustfmt will complain about the visual indent—and rightly so! :P

Reply:
Yeah, I use stable rust and forgo to disable it before modifying the code.

Looking forward to rustfmt and clippy being merged.